ivy/functional/backends/torch/elementwise.py

```diff
@@ -48,7 +50,8 @@ def bitwise_xor(
 bitwise_xor.support_native_out = True
 
 
-@with_unsupported_dtypes({"2.0.1 and below": ("float16", "complex")}, backend_version)
```

Comment:
Could you please revert the changes made to `with_unsupported_dtypes` in this file?